sale_packaging_default/models/sale_order_line.py

```diff
@@ -22,6 +22,9 @@ def onchange(self, values, field_name, field_onchange):
     @api.depends("product_id", "product_uom_qty", "product_uom")
     def _compute_product_packaging_id(self):
         """Set a default packaging for sales if possible."""
+        if self.env.context.get("bypass_default_packaging", False):
```

Comment:
issue: please [avoid negative booleans](https://www.recallstack.icu/en/2020/09/01/do-not-use-negative-booleans-please/)

```suggestion
        if not self.env.context.get("apply_default_packaging", True):
```

Reply:
Thank @yajo  for your suggestion